package.json

```diff
@@ -28,47 +29,45 @@
     "url": "https://github.com/DomainGroupOSS/react-media-query-hoc/issues"
   },
   "homepage": "https://github.com/DomainGroupOSS/react-media-query-hoc#readme",
-  "eslintConfig": {
-    "extends": "airbnb",
-    "rules": {
-      "react/jsx-filename-extension": "off"
-    }
-  },
   "babel": {
     "presets": [
-      "es2015",
-      "stage-0",
-      "react"
+      "react-app"
+    ],
+    "plugins": [
+      "transform-es2015-modules-commonjs",
+      "transform-runtime"
     ]
   },
   "peerDependencies": {
-    "react": "^15.1.0"
+    "react": "^15.1.0 || ^16.0.0"
   },
   "devDependencies": {
     "babel-cli": "^6.10.1",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-0": "^6.5.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-react-app": "^3.1.1",
     "babel-register": "^6.9.0",
     "chai": "^4.1.2",
-    "enzyme": "^2.3.0",
-    "eslint": "^3.19.0",
-    "eslint-config-airbnb": "^15.0.1",
-    "eslint-plugin-import": "^2.3.0",
-    "eslint-plugin-jsx-a11y": "^5.0.3",
-    "eslint-plugin-react": "^7.0.1",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "eslint": "^4.19.1",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-plugin-import": "^2.11.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-react": "^7.8.2",
     "jsdom": "^11.0.0",
     "match-media-mock": "^0.1.0",
-    "mocha": "^2.5.3",
-    "proxyquire": "^1.8.0",
-    "react": "^15.1.0",
+    "mocha": "^5.2.0",
+    "react": "^16.3.2",
     "react-addons-test-utils": "^15.1.0",
-    "react-dom": "^15.1.0",
-    "sinon": "^3.2.1"
+    "react-dom": "^16.3.2",
+    "sinon": "^5.0.7"
   },
   "dependencies": {
-    "hoist-non-react-statics": "^1.2.0",
-    "matchmedia": "^0.1.2",
+    "babel-runtime": "^6.26.0",
```

Comment:
I dont think this should be a dependancy - see docs:
https://babeljs.io/docs/plugins/transform-runtime/ "The transformation plugin is typically used only in development, but the runtime itself will be depended on by your deployed/published code. See the examples below for more details." - so it depends on use of this library 
We at domain import this separately on server

Can you clarify why its needed with your changes ?

Reply:
> but the runtime itself will be depended on by your deployed/published code

The transpiled code requires `babel-runtime`, it provides the polyfills so we don't have to worry about the consumers browser version and if they have `babel-polyfill`. It's awesome for libraries, the Babel docs say:

> This plugin is recommended in a library/tool.